docs/src/components/DocsBehaviorRoot.tsx

```diff
@@ -0,0 +1,72 @@
+import * as _ from 'lodash'
+import PropTypes from 'prop-types'
+import * as React from 'react'
+const behaviorMenuItems = require('docs/src/componentMenuBehaviors')
+import { Grid } from 'semantic-ui-react'
+import ComponentExampleTitle from './ComponentDoc/ComponentExample/ComponentExampleTitle'
+class DocsBehaviorRoot extends React.Component<any, any> {
+  static propTypes = {
+    children: PropTypes.node,
+    match: PropTypes.shape({
+      params: PropTypes.shape({
+        name: PropTypes.string.isRequired,
+      }),
+    }),
+  }
+
+  getNameFromFileName(fileName: string) {
+    const divided = _.startCase(fileName.replace('ts', ''))
```

Comment:
it is better to introduce more descriptive name for this variable (like `baseName`)